src/if-run/lib/compute.ts

```diff
@@ -77,6 +78,10 @@ const computeNode = async (node: Node, params: ComputeParams): Promise<any> => {
   inputStorage = mergeDefaults(inputStorage, defaults);
   const pipelineCopy = structuredClone(pipeline) || {};
 
+  if (Object.keys(pipelineCopy).length === 0) {
```

Comment:
the `Object.keys(pipelineCopy).length` has `[ '0' ]` if there isn't compute, observe or regroup in the manifest. You can test `manifests/examples/builtins/sum/success.yml`
by commenting `compute` and fixing the tabulation